JsonLD VC BBS Go (#280)
hyperledger-labs/hyperledger-labs.github.io: labs/jsonld-vc-bbs-go.md

---
layout: default
title: JsonLD VC BBS Go
parent: Labs
---
# Lab Name
JsonLD VC BBS Go

# Short Description
Golang implementation of W3C JSON-LD BBS credentials

# Scope of Lab

> This library fits into The Linux Foundation Decentralized-Trust branch. It implements issuance, verification, proof generation of [W3C JSON-LD](https://decentralized-id.com/web-standards/w3c/verifiable-credentials/data-integrity-bbs+/) credentials in Golang. The initially supported standards are  `BbsBlsSignature2020` and `BbsBlsSignatureProof2020`. Later we will add new standards as they are adopted.

Reviewer: ```suggestion
This library implements issuance, verification, proof generation of [W3C JSON-LD ZKP with BBS+ Verifiable Credentials](https://decentralized-id.com/web-standards/w3c/verifiable-credentials/data-integrity-bbs+/) in Golang. The initially supported standards are `BbsBlsSignature2020` and `BbsBlsSignatureProof2020`. Later we will add new standards as they are adopted.
```

Author: Changed the description
